app/assets/stylesheets/govspeak/_highlights.scss

```diff
@@ -0,0 +1,6 @@
+.information {
+  border-style: solid;
+  border-radius: 9px;
+  padding: 40px;
+  margin: 50px;
+}
```

Comment:
I think we should choose how we organise styles and stick with it - I am happy with putting styles for each govspeak component into their own css directory, like this one.

Although, I think it should be 
```
. gem-c-govspeak {
  .information {...}
}
```

So that if we have a website `information` class we don't get the govspeak styling in there.

Reply:
no problem, will separate them out and do tables as well